.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Toolbox",
```

Comment:
Would it be useful to give this is a more specific name, this would be okay for now but it might help if other scripts get thrown in here. 

Something like 'Run debug server'? 

Unsure on what's the clearest

Context for where the name comes up
<img width="448" alt="Screenshot 2021-04-12 at 14 39 54" src="https://user-images.githubusercontent.com/2844572/114403851-0cb08d80-9b9d-11eb-80e9-d421d2891cb9.png">